CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ find_package(catkin REQUIRED COMPONENTS
   moveit_core
   roscpp
   pluginlib
+  opw_kinematics
```

Comment:
`opw_kinematics` is not a Catkin package. It's a plain CMake package, so you'd need to do something like:

```cmake
find_package(opw_kinematics REQUIRED)

...

target_link_libraries(${MOVEIT_LIB_NAME}
  ${catkin_LIBRARIES}
  opw_kinematics::opw_kinematics
)
```

Reply:
Yep indeed, thanks for the pointer! I was not sure about the namespace thing `opw_kinematics::opw_kinematics`.